tests/testthat/test-twosampletests.R

```diff
@@ -0,0 +1,192 @@
+context("Two sample tests")
+
+tb.AgeM <- structure(c(11.6455696202532, 11.6455696202532, 10.8860759493671,
```

Comment:
this file would be easier to read if all these `dput`/`structure` calls were removed and instead the variables were just loaded from an `rda` file in `tests/testthat` or `inst/testdata`.

Reply:
I think these output tables are pretty small, so it shouldn't affect readability that much. The other reason I didn't move them is because sometime I have p-value from a Q table and I might want to do a search to see if any values in the testthat file match that p-value